tests/algorithm/test_quicksort.py

```diff
@@ -12,3 +12,8 @@ def test_quicksort_inplace():
     input = [23, 12.5, 3, 32, -3, -100, 0, 4.25, 5, 1, 83]
     quicksort(input, low=3, high=7)
     assert input == [23, 12.5, 3, -100, -3, 0, 4.25, 32, 5, 1, 83]
+
+
+def test_bubblesort():
```

Comment:
Maybe we can consolidate this test module by renaming it `test_sort.py` and use same input across all tests for sorting algorithms.

Reply:
Yes. Thanks for pointing this out.

Reply:
Done